docs/modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -118,6 +118,21 @@ owncloud --logdir /tmp/owncloud_logs --logexpire 48
 
 Adding the `--logdebug` flag increases the verbosity of the generated log files.
 
+==== Logging in the Console
+
+If the ownCloud client isn't able to start and immediately crashes the first two options are not available. 
+Therefore it might need to be necessary to start the ownCloud client using the command line in order to be see the error message
+
+On Linux and Mac simply open the terminal and run `owncloud`.
+
+On Windows open a PowerShell and run the following command:
```

Comment:
--logfile - --logflush is also recommended on unix we will just get a more detailed log.

--logdebug will create a more detailed log.

The pipe on Windows is required as usually no console is attached to a gui process so the output is lost.

Reply:
Thanks for your input!
Better like that?